casts/cards/src/UICollectionCard/types.ts

```diff
@@ -40,4 +44,7 @@ export type BadgeProps = {
 export type CollectionSlideProps = {
     content: MediaCardContent;
     style: ViewStyle;
+    isVisible: boolean;
+    onLoad: () => void;
+    onError: () => void;
```

Comment:
How about returning some `Error` instance here? (it's a good tone to provide an error in such methods)